f5/bigip/tm/sys/test/functional/test_provision.py

```diff
@@ -17,7 +17,7 @@
 class TestProvision(object):
     def test_Provision(self, request, bigip):
         # Load
-        ltmprov = bigip.sys.provision.ltm.load()
+        ltmprov = bigip.tm.sys.provision.ltm.load()
```

Comment:
Bigip fixture is deprecated and it is better to use mgmt_root, with that said we I believe we have changed the fixture so it is a child of mgmt_root:

https://github.com/F5Networks/f5-common-python/blob/development/f5sdk_plugins/fixtures.py#L102

so you can remove the TM.